host/sgx/sgxload.c

```diff
@@ -402,9 +439,19 @@ oe_result_t oe_sgx_create_enclave(
                 enclave_error);
 
         secs->base = (uint64_t)base;
```

Comment:
This line can be placed to the else branch of the if (context->create_zero_base_enclave) --- either set the secs->base as base or 0.

Reply:
done